helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -30,24 +30,32 @@ csiResizerTag: v0.3.0
 
 # csi-driver tag
 csiDriverImage: hpestorage/csi-driver
-csiDriverTag: 1.1.0-beta
+csiDriverTag: v1.0.0-beta
 
 # csp-hpe
 cspImage: hpestorage/nimble-csp
-cspTag: 1.1.0-beta
+cspTag: v1.0.0-beta
+cspName: nimble-csp
+
+# csp-hpe for 3par
+cspImage: hpestorage/csp_3par_primera
```

Comment:
i think we can introduce a variable called as "storagePlatform" and have "nimble" or "hpe3parprimera" as possible values. That way we don't need to add duplicate variables.

Then we don't need variables like cspName. we can derive like {{ .Values.storagePlatform }}-csp and secret like {{ .Values.storagePlatform }}-secret directly.